### ISSUE TITLE
Can't resolve 'prettier-plugin-astro'

### ISSUE THREAD

**Issue by denchen** [tailwindlabs/prettier-plugin-tailwindcss#319]
**What version of `prettier-plugin-tailwindcss` are you using?**

0.6.7

**What version of Tailwind CSS are you using?**

3.4.13

**What version of Node.js are you using?**

v20.12.2

**What package manager are you using?**

npm

**What operating system are you using?**

macOS

**Reproduction URL**

**Describe your issue**

Running prettier as part of my pre-commit chain, I get:

```
✖ prettier --single-quote --write:
[error] Can't resolve 'prettier-plugin-astro' in '/Users/denchen/git/ui/node_modules/prettier-plugin-tailwindcss/dist/index.mjs'
```

**Comment by idimetrix** [comment]
The same problem.

**Comment by lstephensca** [comment]
I have a very similar issue in a new nextjs 14 project I started. However, I am getting `Can't resolve @shopify/prettier-plugin-liquid`.

**Comment by scrlkx** [comment]
I started to have a similar problem on an Astro project right after updating to 0.6.7 (0.6.6 works well).

```bash
npx prettier src/ --check

Checking formatting...
[error] Can't resolve '@shopify/prettier-plugin-liquid' in '../node_modules/prettier-plugin-tailwindcss/dist/index.mjs'
```

**Comment by thecrypticace** [comment]
Sorry fix incoming…

**Comment by worapolburaphan** [comment]
downgrade to 0.6.6 is work for me now @denchen

**Comment by thecrypticace** [comment]
Released as v0.6.8 — apologies. Need to tweak the test suite to detect this case correctly.
